src/main/kotlin/no/nav/familie/ef/sak/vedtak/dto/VedtakBarnetilsynDto.kt

```diff
@@ -18,16 +18,8 @@ data class TilleggsstønadDto(val harTilleggsstønad: Boolean,
                              val perioder: List<PeriodeMedBeløpDto> = emptyList(),
                              val begrunnelse: String?)
 
-data class BarnetilsynperiodeDto(val årMånedFra: YearMonth,
```

Comment:
Hører ikke disse hjemme her i stedet for i beregning da? Og så bruker man beregningService for å beregne perioder som man vedtar?